packages/api/cms-api/src/kubernetes/kubernetes.service.ts

```diff
@@ -68,7 +68,7 @@ export class KubernetesService {
     }
 
     async deleteJob(name: string): Promise<void> {
-        const { response } = await this.batchApi.deleteNamespacedJob(name, this.namespace);
+        const { response } = await this.batchApi.deleteNamespacedJob(name, this.namespace, undefined, undefined, undefined, undefined, "Background");
```

Comment:
In the commit description `"background"` is lowercase, here it is uppercase – is this correct?

Reply:
According to the description of the function in the package, it should be uppercase.